libs/utils/utils.js

```diff
@@ -739,16 +739,14 @@ async function checkForPageMods() {
       .filter((path) => path?.trim());
   }
 
-  const { mep: mepOverride } = Object.fromEntries(PAGE_URL.searchParams);
   const { env } = getConfig();
-  const previewPage = env?.name === 'stage' || env?.name === 'local';
-  if (mepOverride || mepOverride === '' || previewPage) {
+  const mep = PAGE_URL.searchParams.get('mep');
+  if (mep !== null || (env.name !== 'prod' && (persEnabled || targetEnabled))) {
```

Comment:
Seems like we got some unit tests failing @chrischrischris `env?..name` or need to properly set the configs for the failing tests

> Rich Results > add the NewsArticle rich results
      TypeError: Cannot read properties of undefined (reading 'name')
        at checkForPageMods (libs/utils/utils.js:744:28)
        at loadArea (libs/utils/utils.js:844:11)
        at o.<anonymous> (test/utils/richresults.test.js:9:11)